tests/serverDescription/serverDescription-getType-001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\ServerDescription::getType()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$expected_types = array(
+    MongoDB\Driver\ServerDescription::TYPE_STANDALONE,
+    MongoDB\Driver\ServerDescription::TYPE_MONGOS,
+    MongoDB\Driver\ServerDescription::TYPE_RS_PRIMARY
+);
+
+function isExpectedType(int $type, array $expected_types) {
+    foreach ($expected_types as $expected_type) {
+        if ($expected_type == $type) {
```

Comment:
Whenever possible, use the strict equality operator. See https://www.php.net/manual/en/language.operators.comparison.php for more info.
```suggestion
        if ($expected_type === $type) {
```

Reply:
Alternatively, you can replace the entire `isExpectedType` function with [`in_array`](https://www.php.net/in_array) and use the strict option. That would let you inline a call to `in_array` in the `var_dump()` below.